source/write/bulk-write.txt

```diff
@@ -0,0 +1,334 @@
+.. _ruby-bulk-write:
+
+=====================
+Bulk Write Operations
+=====================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+
+.. meta::
+   :keywords: insert, update, replace, code example, multiple changes
+
+Overview
+--------
+
+This guide shows you how to use the {+driver-short+} to perform a bulk
+write operation that makes multiple changes to your data in a single
+database call.
+
+Consider a situation that requires you to insert documents, update
+documents, and delete documents for the same task. If you use
+the individual write methods to perform each type of operation, each write
+accesses the database separately. You can use a bulk write operation to
```

Comment:
S: Simplify and clarify that this page is offering an alternative to doing it this way

```suggestion
documents, and delete documents for the same task. If you use
the individual write methods, each write operation
accesses the database separately. Instead, you can use a bulk write operation to
```